src/microbe_stage/editor/IEditorReportData.cs

```diff
@@ -3,8 +3,20 @@
 /// </summary>
 public interface IEditorReportData : IEditor
 {
+    /// <summary>
+    ///   Returns the patch the player is currently in. If the player wants to move, returns the same
+    ///   as <see cref="TargetPatch"/>
+    /// </summary>
     public Patch CurrentPatch { get; }
 
+    /// <summary>
+    ///   Returns the patch the player wants to move after editing
+    /// </summary>
+    public Patch? TargetPatch { get; }
+
+    /// <summary>
+    ///   Returns the patch the player selected in the patch map
```

Comment:
```suggestion
    ///   Returns the patch the player selected in the patch map (for displaying in the GUI)
```
I think this clarification would be good.